python/ray/util/sgd/torch/training_operator.py

```diff
@@ -151,15 +151,22 @@ def _set_timers(self, timers):
         """Passes in the timers from the Runner."""
         self.timers = timers
 
-    def _configure_amp(self, amp, models, optimizers, apex_args):
+    def _configure_apex_amp(self, amp, models, optimizers, apex_args):
         models, optimizers = amp.initialize(models, optimizers, **apex_args)
         return models, optimizers
 
     def _configure_ddp(self, models, device_ids, ddp_args):
-        return [
+        models = [
             DistributedDataParallel(model, device_ids=device_ids, **ddp_args)
             for model in models
         ]
+        if self.use_fp16_native:
+            # this is hacky but that's how pytorch lighting does it
```

Comment:
This can be removed right?

Reply:
It's been reverted in the second commit, not sure why it's still showing like this